src/odemis/gui/model/__init__.py

```diff
@@ -347,7 +347,9 @@ def __init__(self, microscope):
             elif self.role == "meteor":
                 required_roles += ["light", "stage", "focus"]
             elif self.role in ("sparc", "sparc2"):
-                required_roles += ["e-beam", "mirror", "lens"]
+                required_roles += ["e-beam", "mirror"]
```

Comment:
I would add this comment since this is a semi odd configuration. The rest I approve:
`required_roles += ["e-beam", "mirror"]  # SPARCv1 can also work without a lens.`